### PR TITLE
[DM-3529] Create CL note

### PR DESCRIPTION
**Pull request by bozhidardedov**

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3529
+version: ""
+---
+
+The deprecated hookDeviceGridAction has been removed. Instead the recently introduced hookDataGridActionControls should be used. It gives more control over data grid actions allowing actions to be added to any data grid, to override existing actions, etc.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md:18]:
```suggestion
The deprecated `hookDeviceGridAction` has been removed. Instead the recently introduced `hookDataGridActionControls` should be used. It gives more control over data grid actions, such as allowing actions to be added to any data grid or to override existing actions.
```

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Deprecated hookDeviceGridAction has been removed
+product_area: Device management & connectivity
+change_type:
+  - value: change-3BQrQ6adS
+    label: API change
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md:10]: The component does not match the product area.

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Deprecated hookDeviceGridAction has been removed
+product_area: Device management & connectivity
+change_type:
+  - value: change-3BQrQ6adS
+    label: API change
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md:7]: Should be either Fix or Announcement

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Deprecated hookDeviceGridAction has been removed
+product_area: Device management & connectivity
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md:4]: Should probably be Application enablement & solutions

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md (hunk 3 above)
Comment (bozhidardedov) [reply content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md]: Changed.

File: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md (hunk 4 above)
Comment (bozhidardedov) [reply content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md]: Changed.

**Review by dafkk — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**